src/test/unit/DelegationUnit.t.sol

```diff
@@ -47,275 +40,459 @@ contract DelegationUnitTests is EigenLayerTestHelper {
     bytes32 emptySalt;
 
     // reused in various tests. in storage to help handle stack-too-deep errors
-    address _operator = address(this);
+    address defaultStaker = cheats.addr(uint256(123_456_789));
+    address defaultOperator = address(this);
```

Comment:
In the middleware tests, I've been using a different pattern than the "default X" type thing we do everywhere.

My test setups generate new users on demand, and just increment addresses/ids in state as needed. (see [state vars here](https://github.com/Layr-Labs/eigenlayer-middleware/blob/a6baa25ef060b53a46dc7c77b769d8711fbe94dc/test/unit/StakeRegistryUnit.t.sol#L66-L72) and [helper method here](https://github.com/Layr-Labs/eigenlayer-middleware/blob/test/stake-registry-tests/test/unit/StakeRegistryUnit.t.sol#L188-L196))

I like my setup when I want to test things that require more than one user - it's very easy to just grab a new user/quorum/etc